docs/guides/kernel.md

```diff
@@ -23,8 +23,10 @@ If your distro is not one of the distros with documentation on this Wiki, you ma
 
 ```bash
 mkdir build && cd build
-git clone --depth=1 https://github.com/t2linux/linux-t2-patches patches
-
+git clone https://github.com/t2linux/linux-t2-patches patches
```

Comment:
Remove this change. It's a temporary thing and will automatically get fixed when 6.10 gets released